src/main/kotlin/org/rust/ide/refactoring/extractTrait/RsExtractTraitProcessor.kt

```diff
@@ -87,12 +95,24 @@ class RsExtractTraitProcessor(
         }
     }
 
-    private fun insertImpls(traitImpl: RsImplItem, trait: RsTraitItem) {
-        impl.parent.addAfter(trait, impl)
+    private fun insertImpls(traitImpl: RsImplItem, trait: RsTraitItem): RsTraitItem {
+        val insertedTrait = impl.parent.addAfter(trait, impl) as RsTraitItem
         impl.parent.addAfter(traitImpl, impl)
         if (impl.members?.childrenOfType<RsItemElement>()?.isEmpty() == true) {
             impl.delete()
         }
+        return insertedTrait
+    }
+
+    private fun addTraitImports(usages: Array<out UsageInfo>, trait: RsTraitItem) {
+        val mods = usages.mapNotNullTo(hashSetOf()) {
+            (it.element as? RsElement)?.containingMod
+        }
+        val elementsToImport = hashSetOf(trait)
+        for (mod in mods) {
+            val context = mod.childOfType<RsElement>() ?: continue
+            RsImportHelper.importElements(context, elementsToImport)
```

Comment:
[minor] It looks like all the public usages of `RsImportHelper#importElements` pass a singleton set. Probably it could be better to extract a public method `importElement` instead